src/main/main.ts

```diff
@@ -204,6 +198,16 @@ app.on('window-all-closed', () => {
 	}
 });
 
+// Open urls in the user's browser
+app.on('web-contents-created', (event, contents) =>{
+	contents.setWindowOpenHandler(({ url }) => {
+		setImmediate(() => {
+			shell.openExternal(url);
+		})
+		return {action: 'deny'};
+	})
```

Comment:
i'm a little concerned about how this will act for SSO signin, but will just try this anyway! thanks for PR!